source/plot/SvgPlot.ooc

```diff
@@ -96,7 +99,7 @@ SvgPlot: class {
 					datasets[j] color = datasets[j] colorBgra svgRGBToString()
 					datasets[j] opacity = ((datasets[j] colorBgra svgRGBAlpha()) as Float) / 255.0f
 				} else {
-					datasets[j] color = colorList[this colorCount % this colorList count]
+					datasets[j] color = colorList[this colorCount % this colorList count] clone()
```

Comment:
Each `dataset` will free its `color` (and colorList entries are shared), so need to copy it.